examples/acl/src/backend/app/views/posts.clj

```diff
@@ -0,0 +1,25 @@
+(ns views.posts
+  (:require
+    [xiana.core :as xiana]))
+
+(defn post-view
+  [{response :response-data :as state}]
+  (xiana/ok
+    (assoc state
+      :response
+      {:status  200
+       :headers {"Content-Type" "text/plain"}
```

Comment:
Is there any reason not to return application/json here?
It would spare you regex parsing
```
(def post-id-regex
  #":posts\/id #uuid \"([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12})")

(defn post-ids
  [body]
  (map last (re-seq post-id-regex body)))
```